requirements/edx/stanford.txt

```diff
@@ -1,14 +1,16 @@
-xblock-image-modal==0.3.1
+xblock-image-modal==0.3.1 # Upload release to PyPI with SHA: 01d998a74e813db99abbb3a5bb0c23a8f4606f19
 -e git+https://github.com/openlearninginitiative/xblock-image-coding.git@cee23e3d59943066159ca13839c82a12d9ccf749#egg=xblock_image_coding
--e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@701788bd1a81ef6d97dd5ef6b5c35998266c62a5#egg=xblock-qualtrics-survey
--e git+https://github.com/Stanford-Online/xblock-free-text-response@6349ab4425818c40e7e603b88953bdf7d869bf50#egg=xblock-free-text-response
--e git+https://github.com/Stanford-Online/xblock-grademe.git@v0.1.0#egg=grademebutton
+-e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@08899b3927a52218b8d42f2c2e98789ccef97faf#egg=xblock-qualtrics-survey
+-e git+https://github.com/Stanford-Online/xblock-free-text-response@c3a802da2609b40768c35b35e5b7cfaac677cd9f#egg=xblock-free-text-response
+# Note: previously the xblock-grademe entry was pulling the 'v0.1.0' version, now it's pointing to the latest commit.
+-e git+https://github.com/Stanford-Online/xblock-grademe.git@ee095e9c06f6f55d4c8b2dbf3efc8424fb650a53#egg=grademebutton
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@32d52edaa820dbdbf846d8a84f6bccbf0b9c8218#egg=xblock_ooyala_player-master
 -e git+https://github.com/Stanford-Online/xblock-submit-and-compare.git@86906db85b804b9f1762f75de6c1582eb6c819f6#egg=xblock-submit-and-compare
 -e git+https://github.com/Stanford-Online/xblock-mufi.git@ee853b8a7668a87c27d13d55c0d149a04b8657b8#egg=xblock_mufi-master
 -e git+https://github.com/Stanford-Online/edx-analytics-data-api-client.git@1b8260ce8dd6edb999fffc46b09f77c83f87e1f9#egg=edx-analytics-data-api-client
 -e git+https://github.com/openlearninginitiative/xblock-inline-dropdown.git@d4b3630838f8df0f2d3706128366650d3b0f158b#egg=inline_dropdown
--e git+https://github.com/Stanford-Online/xblock-in-video-quiz.git@release/v0.1.2#egg=invideoquiz_xblock
+# Note: previously the xblock-in-video-quiz entry was pulling the 'release/v0.1.2' version, now it's pointing to the latest commit. 
```

Comment:
I don't think we need to keep either of the comments.  The information can be seen from the diff and they would need to be removed anyway in the next sha update.